src/main/java/nextstep/courses/domain/Image.java

```diff
@@ -0,0 +1,45 @@
+package nextstep.courses.domain;
+
+public class Image {
+  private static final long MAX_SIZE_IN_BYTES = 1_000_000L;
+  private static final int MIN_WIDTH_IN_PIXEL = 300;
+  private static final int MIN_HEIGHT_IN_PIXEL = 200;
+
+  private Long id;
```

Comment:
```suggestion
  private final Long id;
```
클래스 필드들에 전반적으로 불변하게 유지해보면 어떨까요? 😄

Reply:
https://github.com/next-step/java-lms/pull/717/commits/5a1157ceb16c4af73544f80787f53933ed321a5b 전반적으로 final을 적용하게끔 수정해보았습니다 :)